src/helpers/fromHex.js

```diff
@@ -0,0 +1,15 @@
+import { toUtf8, toDecimal, toAscii } from 'web3-utils'
+export default () =>
+  /**
+   * Returns the string representation of a given hex value
+   *
+   * @param {string} hex The hex string
+   * @param {string} [to='utf8'] The type to convert the hex from (supported types: 'utf8', 'number', 'decimal')
+   * @return {radspec/evaluator/TypedValue}
+   */
+  (hex, to = 'utf8') => {
+    return {
```

Comment:
Little style thing: you can get rid of this return by using the () => ({ key: value }) syntax

Reply:
Done (also for other functions with the same issue).